### ISSUE TITLE
[Docker] /phoronix-test-suite/phoronix-test-suite: No such file or directory

### ISSUE THREAD

**Issue by firejox** [phoronix-test-suite/phoronix-test-suite#353]
I pull the official PTS image but I cannot run it correctly. Here is my commands.
```
[firejox@myhostname ~]$ uname -a
Linux myhostname 5.0.7-arch1-1-ARCH #1 SMP PREEMPT Mon Apr 8 10:37:08 UTC 2019 x86_64 GNU/Linux                                                                              
[firejox@myhostname ~]$ docker pull phoronix/pts                                                                                                                             
Using default tag: latest
latest: Pulling from phoronix/pts
be71862069d7: Pull complete
Digest: sha256:060fc6bc691f7dac286c6a092a620f3765a3f44c52d39cd4962ac977c8deb8f2
Status: Downloaded newer image for phoronix/pts:latest
[firejox@myhostname ~]$ docker run -it  phoronix/pts                                                                                                                         
/phoronix-test-suite/phoronix-test-suite: line 247: dirname: No such file or directory
/phoronix-test-suite/phoronix-test-suite: line 255: cat: No such file or directory
[firejox@myhostname ~]$
```

where can I find **/phoronix-test-suite/phoronix-test-suite** ?

**Comment by michaellarabel** [comment]
Not sure if you somehow corrupted your Docker image or what, but that path should be there as part of the default image. I just did a "docker run -it phoronix/pts" and it fired up just fine without problems.

**Comment by firejox** [comment]
It is still not working even I clean whole cache and repull the image.
```
[firejox@myhostname ~]$ docker system prune -a
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all images without at least one container associated to them
        - all build cache
Are you sure you want to continue? [y/N] y
Total reclaimed space: 0B
[firejox@myhostname ~]$ docker pull phoronix/pts                                                                                                                             
Using default tag: latest
latest: Pulling from phoronix/pts
be71862069d7: Pull complete
Digest: sha256:060fc6bc691f7dac286c6a092a620f3765a3f44c52d39cd4962ac977c8deb8f2
Status: Downloaded newer image for phoronix/pts:latest
[firejox@myhostname ~]$ docker run -it phoronix/pts
/phoronix-test-suite/phoronix-test-suite: line 247: dirname: No such file or directory
/phoronix-test-suite/phoronix-test-suite: line 255: cat: No such file or directory
```

This is my docker system event
```
[firejox@myhostname develop]$ docker system events
2019-06-02T09:31:12.484667528+08:00 container create 6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433 (image=phoronix/pts:latest, name=festive_hertz)        
2019-06-02T09:31:12.486380547+08:00 container attach 6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433 (image=phoronix/pts:latest, name=festive_hertz)        
2019-06-02T09:31:12.863761583+08:00 network connect cff7e211a05ef22e9047ab225f090b3be476549de0fb622bb985a1c1c8037433 (container=6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433, name=bridge, type=bridge)
2019-06-02T09:31:13.710930633+08:00 container start 6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433 (image=phoronix/pts:latest, name=festive_hertz)         
2019-06-02T09:31:13.712788055+08:00 container resize 6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433 (height=39, image=phoronix/pts:latest, name=festive_hertz, width=174)
2019-06-02T09:31:14.184583344+08:00 container die 6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433 (exitCode=127, image=phoronix/pts:latest, name=festive_hertz)
2019-06-02T09:31:14.512400175+08:00 network disconnect cff7e211a05ef22e9047ab225f090b3be476549de0fb622bb985a1c1c8037433 (container=6bc7538bc5170e06702566ad7d27cfb3fddfe2e954f14f198afdd3aea1131433, name=bridge, type=bridge)
```

And This is my docker system info
```
[firejox@myhostname ~]$ docker system info
Containers: 4
 Running: 0
 Paused: 0
 Stopped: 4
Images: 1
Server Version: 18.09.5-ce
Storage Driver: devicemapper
 Pool Name: docker-8:5-524729-pool
 Pool Blocksize: 65.54kB
 Base Device Size: 10.74GB
 Backing Filesystem: xfs
 Udev Sync Supported: true
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Data Space Used: 7.594GB
 Data Space Total: 107.4GB
 Data Space Available: 54.87GB
 Metadata Space Used: 21.62MB
 Metadata Space Total: 2.147GB
 Metadata Space Available: 2.126GB
 Thin Pool Minimum Free Space: 10.74GB
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: true
 Deferred Deleted Device Count: 0
 Library Version: 1.02.156 (2019-03-22)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 894b81a4b802e4eb2a91d1ce216b8817763c29fb.m
runc version: 69ae5da6afdcaaf38285a10b36f362e41cb298d6
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 5.0.7-arch1-1-ARCH
Operating System: Arch Linux
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 15.66GiB
Name: myhostname
ID: RGQF:PQXF:3UCJ:AJCD:P6IE:OCJJ:JUXY:ZYMH:YKMB:GRQS:SJYW:XVSC
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Username: firejox
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: the devicemapper storage-driver is deprecated, and will be removed in a future release.
WARNING: devicemapper: usage of loopback devices is strongly discouraged for production use.
         Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
```

**Comment by jmaerki** [comment]
Same problem here.

**Comment by michaellarabel** [comment]
An overhauled version of the `phoronix/pts` Docker image is now available with PTS 9.8.
